client/reader/stream/reader-sidebar-organizations/list.jsx

```diff
@@ -0,0 +1,65 @@
+import { localize } from 'i18n-calypso';
+import { map } from 'lodash';
+import PropTypes from 'prop-types';
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import AutomatticLogo from 'calypso/assets/images/icons/a8c-logo.svg';
+import P2Logo from 'calypso/assets/images/icons/p2-logo.svg';
+import SVGIcon from 'calypso/components/svg-icon';
+import { toggleReaderSidebarOrganization } from 'calypso/state/reader-ui/sidebar/actions';
+import { isOrganizationOpen } from 'calypso/state/reader-ui/sidebar/selectors';
+import getOrganizationSites from 'calypso/state/reader/follows/selectors/get-reader-follows-organization';
+import { AUTOMATTIC_ORG_ID } from 'calypso/state/reader/organizations/constants';
+import ReaderSidebarOrganizationsListItem from './list-item';
+import '../style.scss';
+
+export class ReaderSidebarOrganizationsList extends Component {
```

Comment:
It's weird we have two `ReaderSidebarOrganizationsList` now. Feels like something is off and we could further deduplicate logic. Or if we have two different version of these components for some reason, we should rename them and make the difference more transparent.